.circleci/config.yml

```diff
@@ -69,37 +69,37 @@ jobs:
       - run: make setup
       - run: curl -sL https://raw.githubusercontent.com/goreleaser/get/master/get | bash
       - setup_remote_docker
-      - run: make build_release_image
-      - run: make push_release_image
+      - run: make build-release-image
+      - run: make push-release-image
       - store_artifacts:
           path: dist
 
 workflows:
   version: 2
   build_and_push:
     jobs:
-      - build_server:
-          filters:
-            tags:
-              only: /.*/
-      - build_ui:
-          filters:
-            tags:
-              only: /.*/
-      - push_master_image:
-          requires:
-            - build_server
-            - build_ui
-          filters:
-            branches:
-              only:
-                - master
-      - push_release_image:
-          requires:
-            - build_server
-            - build_ui
-          filters:
-            tags:
-              only: /\d{1,2}\.\d{1,2}\.\d{1,2}/
-            branches:
-              ignore: /.*/
\ No newline at end of file
+    - build_server:
+        filters:
+          tags:
+            only: /.*/
+    - build_ui:
+        filters:
+          tags:
+            only: /.*/
+    - push_master_image:
+        requires:
+        - build_server
```

Comment:
Should this not be indented?

Reply:
Just checked the CircleCI docs - the indentation levels in the previous file [are what they suggest](https://circleci.com/docs/2.0/configuration-reference/#section=configuration), for that reason I would suggest we don't change this.

Reply:
it is equals before